typespec-extension/src/code-model-builder.ts

```diff
@@ -279,67 +276,38 @@ export class CodeModelBuilder {
     return this.codeModel;
   }
 
-  private processHost(server: HttpServer | undefined): Parameter[] {
+  private processHostParametersFromSdkType(sdkPathParameters: SdkPathParameter[]): Parameter[] {
```

Comment:
I just put the suffix to make the name more obvious. I can do one-time suffix removal.

Reply:
Also I found typescript does not support a traditional OO language function overloading, which means if you define two functions with the same name but difference parameters and write implementation for each of them. There would be compilation error.
<img width="796" alt="image" src="https://github.com/user-attachments/assets/f70dd892-6613-4bde-9439-f01e34bf8e50">

If you want to do overload in typescript, you need to write code like below- declare the functions and have one implementation.
```ts
function add(a: number, b: number): number;
function add(a: string, b: string): number;

function add(a: unknown, b: unknown): number {
  if (typeof a === "string" && typeof b === "string") {
    return Number(a) + Number(b);
  } else if (typeof a === "number" && typeof b === "number") {
    return a + b;
  }
  return 0;
}
```
references: https://www.telerik.com/blogs/how-implement-function-overloading-typescript
https://stackoverflow.com/questions/13212625/typescript-function-overloading

That's why I name the functions differently to avoid ambiguity and be more obvious.
But I can do a one-time suffix removal if you all prefer. Please let me know your opinion. Thanks!

Reply:
Yeah, I know about the ts overload.

What I mean is there is no `processHost` in your ts file (correct me if wrong). So overload does not take effect (if there were, it is history that not concern the PR).

We'd generally prefer shorter name, if that already get things clear.

Reply:
Sure, I will do a one-time suffix removal.

Reply:
Removed the suffix for process clients/operation and related functions. Did not touch the process schema yet as some helper functions are still using raw type.